samples/esb/esb_prx_ble/README.rst

```diff
@@ -0,0 +1,132 @@
+.. _esb_prx_ble:
+
+Enhanced ShockBurst: Receiver with Bluetooth LE
+################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample shows how to use the :ref:`ug_esb` protocol in receiver mode cooperatively with Bluetooth LE protocol.
+It demonstrates how to configure the Enhanced ShockBurst protocol to receive packets while simultaneously running Bluetooth LE services.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Additionally, if you want to test the Enhanced ShockBurst Transmitter functionality, you need to build and run the :ref:`esb_ptx` sample.
+You can use any two of the listed development kits and mix different development kits.
+
+Overview
+********
+
+The sample consists of one Receiver that uses the :ref:`esb_README` library in combination with Bluetooth LE functionality.
+After building and programming each sample on an nRF52 Series development kit, you can test that packets that are sent by the kit that runs the :ref:`Transmitter <esb_ptx>` sample are picked up by the kit that runs the Receiver sample.
```

Comment:
Is it just for the nRF52 Series or on your development kit as the sample as support for 53 and 54 Series?